cassandra/cqltypes.py

```diff
@@ -1410,6 +1433,11 @@ class VectorType(_CassandraType):
     vector_size = 0
     subtype = None
 
+    @classmethod
+    def serial_size(cls):
+        serialized_size = cls.subtype.serial_size()
+        return cls.vector_size * serialized_size if serialized_size is not None else None
```

Comment:
Here (as in many other points in this PR) we're following the [example of what's done](https://github.com/apache/cassandra/commit/ae537abc6494564d7254a2126465522d86b44c1e#diff-8d59be6f2cd342929e99b6da7f889239e85605cbb11c5b1044a5d04421099486R91-R93) on the server side.